buildspec.yml

```diff
@@ -7,19 +7,19 @@ env:
     AWS_DEFAULT_REGION: eu-central-1
     TEST_REPORT_DIR: "./tests/report"
     TEST_REPORT_FILE: "nose2-junit.xml"
+    PIPENV_NOSPIN: 1
   parameter-store:
     CI_DOCKERHUB_USER: "/dockerhub/user"
     CI_DOCKERHUB_PASSWORD: "/dockerhub/password"
 
-
 phases:
   install:
     runtime-versions:
       docker: 18
     commands:
       - echo "Installing necessary softwares"
       - docker login -u ${CI_DOCKERHUB_USER} -p ${CI_DOCKERHUB_PASSWORD}
-      - apt-get update && apt-get install -y docker-compose python3-pip
+      # - apt-get update && apt-get install -y docker-compose python3 python3-venv python3-pip
```

Comment:
should we remove this line altogether?

Reply:
True this line is currently failing on codebuild which currently failed to install packages (same issue on your service-icons) and it seems that we don't require any package for service alti as python3 is already present. I'll remove the line though